packages/platform-android/native_modules.gradle

```diff
@@ -188,7 +188,7 @@ class ReactNativeModules {
 
     try {
       cmdProcess = Runtime.getRuntime().exec(command, null, root)
-      def bufferedReader = new BufferedReader(new InputStreamReader(cmdProcess.getInputStream()))
+      def bufferedReader = new BufferedReader(new InputStreamReader(cmdProcess.getOutputStream()))
```

Comment:
Why not `OutputStreamReader` instead of `InputStreamReader` then?

Reply:
AFAIK that doesn't exist.

Reply:
I think you mean `OutputStreamWriter`?

Reply:
> Why not `OutputStreamReader` instead of `InputStreamReader` then?

I think you mean `OutputStreamWriter`?

Reply:
Hmm I see the problem with this patch. getInputStream yields the output stream of the subprocess.